README.md

```diff
@@ -78,8 +28,9 @@ Check out the [deployment documentation](https://nuxt.com/docs/getting-started/d
 
 ### Resources
 
-- [Vite](https://vitejs.dev/)
+- [Nuxt](https://nuxt.com/) (for brand icons)
```

Comment:
Probablement un résidu de conflit git, mais le (for brand icons) n'est pas approprié ici